src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $card-bg;
+  padding: 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 0;
+  }
+
+  &__title {
+    color: $card-text-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    color: $card-text-secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    width: 53px;
+    height: 16px;
+    justify-content: flex-end;
+    align-items: center;
+    color: $card-text-secondary;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+
+    &-text {
+      color: $card-text-secondary;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-value {
+      color: $card-text-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__link {
+    display: flex;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    background: $moyo-blue;
+    text-decoration: none;
+    align-items: center;
+    justify-content: center;
+    margin-top: 16px;
+    border: 1px solid transparent;
+
+    color: $card-bg;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+
+    &:hover {
+      background: $card-bg;
+      color: $moyo-blue;
+      border-color: $moyo-blue;
+    }
+  }
+}
```

Comment:
Looks valid

One small hint - rather than altering the color of the border I would just add a `1px solid $moyo-blue` border to the button and don't change it.
As it is the same color as the background it would not be visible. ;)